### PR TITLE
Add label to show isolated group 

### PR DESCRIPTION
**Pull request by nixocio**
Add label to show isolated group.

See: https://tower-mockups.testing.ansible.com/patternfly/instance-groups/instance-groups/

image

### CODE REVIEW

**Review by jakemcdermott — APPROVED** [review]

File: awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupListItem.jsx [hunk 1]
```diff
@@ -108,6 +109,13 @@ function InstanceGroupListItem({
                   <b>{instanceGroup.name}</b>
                 </Link>
               </span>
+              {instanceGroup.is_isolated ? (
+                <span css="margin-left: 12px">
+                  <Chip key={instanceGroup.id} isReadOnly>
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupListItem.jsx]: @nixocio I think the `Label` component is a better choice. It more closely align with the mockups:

Screenshot from 2020-08-11 10-09-01

**Review by jakemcdermott — CHANGES_REQUESTED** [review]

File: awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupListItem.jsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { Link } from 'react-router-dom';
 import 'styled-components/macro';
 import {
   Badge as PFBadge,
+  Chip,
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupListItem.jsx]:
```suggestion
  Label,
```

File: awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupListItem.jsx [hunk 3]
```diff
@@ -108,6 +109,13 @@ function InstanceGroupListItem({
                   <b>{instanceGroup.name}</b>
                 </Link>
               </span>
+              {instanceGroup.is_isolated ? (
+                <span css="margin-left: 12px">
+                  <Chip key={instanceGroup.id} isReadOnly>
+                    {i18n._(t`Isolated`)}
+                  </Chip>
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupListItem.jsx]:
```suggestion
                  <Label key={instanceGroup.id}>
                    {i18n._(t`Isolated`)}
                  </Label>
```

Comment (nixocio) [reply awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupListItem.jsx]: I also need to update this in the detail page. Thanks for review @jakemcdermott .

**Comment by nixocio** [comment]
@jakemcdermott, I addressed your comments. Changes were slightly more involving that I anticipated.

**Review by jakemcdermott — APPROVED** [review]

**Comment by nixocio** [comment]
regate

**Pull request merged**